src/app/[locale]/dataset-list.cy.tsx

```diff
@@ -0,0 +1,36 @@
+import DatasetList from './dataset-list';
+
+describe('<DatasetList />', () => {
+  context('List rendering', () => {
+    it('Shows a dataset card if one dataset is returned in the search result', () => {
+      cy.fixture('search-results/one-dataset.json').then(searchResult => {
+        cy.intercept({method: 'GET', url: '/api/datasets*'}, searchResult);
+        cy.mount(
+          <DatasetList initialSearchResult={searchResult} locale="en" />
+        );
+        cy.getBySel('dataset-card').its('length').should('eq', 1);
+      });
+    });
+
+    it('Shows 3 dataset card if 3 datasets are returned in the search result', () => {
+      cy.fixture('search-results/multiple-datasets.json').then(searchResult => {
+        cy.intercept({method: 'GET', url: '/api/datasets*'}, searchResult);
+        cy.mount(
+          <DatasetList initialSearchResult={searchResult} locale="en" />
+        );
+        cy.getBySel('dataset-card').its('length').should('eq', 3);
+      });
+    });
+
+    it('Shows an empty list if 0 datasets are returned in the search result', () => {
```

Comment:
Or: `Shows 0 dataset cards if 0 datasets...` (to be consistent with the other test names)?